javascript/node/selenium-webdriver/firefox/index.js

```diff
@@ -367,14 +373,9 @@ class Driver extends webdriver.WebDriver {
       let service = createGeckoDriverService(binary);
       serverUrl = service.start();
       onQuit = () => service.kill();
+      caps.set(Capability.PROFILE, profile.encode());
```

Comment:
(fwiw if you are making a performance argument about sending profiles over the wire it would be interesting to see a profile that shows it's significant. My expectation would be that on the local machine it's much faster than starting Firefox, for example, so won't make much difference to the overall test time).